src/cow-react/common/pure/RateInfo/index.tsx

```diff
@@ -147,7 +147,7 @@ export function RateInfo({
       )}
       <div>
         <RateWrapper onClick={() => setCurrentIsInversed((state) => !state)}>
-          <span title={currentActiveRate.toSignificant(18) + ' ' + rateInputCurrency.symbol}>
+          <span title={currentActiveRate.toSignificant(18) + ' ' + rateOutputCurrency.symbol}>
```

Comment:
why hardcoded to 18?

Reply:
My bad, must be:
`currentActiveRate.toSignificant(rateOutputCurrency.decimals)`

Reply:
@shoom3301 I will add that